### PR TITLE
English translation and comments corrections

### PR DESCRIPTION
**Pull request by patrickelectric**

### CODE REVIEW

**Comment by patrickelectric** [comment]
Fixes #9 partialy

File: Firmware/ControladorRobo/ControladorRobo.ino [hunk 1]
```diff
@@ -42,7 +42,7 @@ Controller directionController = Controller(0.1, 0, 0, 1/actFreq);
 
 void peripheralsSetup()
 {
-    // Seta os pinos de chave como entrada em pull-up
+    // Set switch pins to PULLUP mode
```
Comment (Williangalvani) [inline Firmware/ControladorRobo/ControladorRobo.ino]: "Sets"? as in "It sets"

File: Firmware/ControladorRobo/ControladorRobo.ino [hunk 2]
```diff
@@ -93,14 +93,10 @@ void loop()
...
-///////////////////////////////////////////////////////////funcoes//////////////////////////////////////////////////////////////////////////////
-
-
-// interrupcao para TIMER1, periodo de amostragem para controle de velocidade
+// TIMER1 interruption, sample time for velocity control
 ISR(TIMER1_COMPA_vect)
 {
-    // Calculo das velocidades dos motores
-
+    // calculus of motor velocity
```
Comment (Williangalvani) [inline Firmware/ControladorRobo/ControladorRobo.ino]: Calculation of motor speed ?
Calculus seems wrong here.
And "velocity" implies a vector.

File: Firmware/ControladorRobo/control.h [hunk 3]
```diff
@@ -14,10 +14,10 @@
 #define umin -1.0
 #define umax 1.0
 
-//Modelo Esquerdo
+// Right model
```
Comment (Williangalvani) [inline Firmware/ControladorRobo/control.h:17]: Right model is kind of vague. This is actually a linear approximation of the system motor + wheel using a "first-order plus deadtime"  (FOPDT) transfer function, where t1 is the time constant, theta is the delay, and kp is the gain.

File: Firmware/ControladorRobo/irarray.h [hunk 4]
```diff
@@ -2,7 +2,7 @@
 
 #include <Arduino.h>
 
-// Passar parte da eeprom para um outra classe
+// TODO: create eeprom crass
```
Comment (Williangalvani) [inline Firmware/ControladorRobo/irarray.h]: I don't think we need an eeprom "crass"

File: Firmware/ControladorRobo/irarray.h [hunk 5]
```diff
@@ -21,15 +21,15 @@ class IRArray
     void saveCalibrationToEEPROM();
     void loadCalibrationFromEEPROM();
 
-    // pinos para leitura analógica
+    // pins to analog read
```
Comment (Williangalvani) [inline Firmware/ControladorRobo/irarray.h]: for ?

File: Firmware/ControladorRobo/irarray.h [hunk 6]
```diff
@@ -21,15 +21,15 @@ class IRArray
...
     static uint16_t m_sensorPin[NUMBER_OF_SENSORS];
     uint8_t m_sensorLow[NUMBER_OF_SENSORS];
     uint8_t m_sensorHigh[NUMBER_OF_SENSORS];
     byte m_enablePin;
     uint16_t m_sensorRaw[NUMBER_OF_SENSORS];
     uint16_t m_sensor[NUMBER_OF_SENSORS];
 
-    // le os sensores e salva o valor lido de cada um no byte[9]
+    // realize calibration routine
```
Comment (Williangalvani) [inline Firmware/ControladorRobo/irarray.h]: "realize"
I know it is technically correct, but it still sounds weird to me.

Comment (patrickelectric) [inline Firmware/ControladorRobo/irarray.h]: ✓

File: Firmware/ControladorRobo/irarray.h (hunk 5 above)
Comment (patrickelectric) [inline Firmware/ControladorRobo/irarray.h]: ✓

File: Firmware/ControladorRobo/irarray.h (hunk 4 above)
Comment (patrickelectric) [inline Firmware/ControladorRobo/irarray.h]: ✓

File: Firmware/ControladorRobo/control.h (hunk 3 above)
Comment (patrickelectric) [inline Firmware/ControladorRobo/control.h:17]: // Linear approximation of motor-wheel system with a first-order equation
what you think ?

File: Firmware/ControladorRobo/ControladorRobo.ino (hunk 2 above)
Comment (patrickelectric) [inline Firmware/ControladorRobo/ControladorRobo.ino]: ✓

File: Firmware/ControladorRobo/ControladorRobo.ino (hunk 1 above)
Comment (patrickelectric) [inline Firmware/ControladorRobo/ControladorRobo.ino]: ✓

File: Firmware/ControladorRobo/control.h (hunk 3 above)
Comment (Williangalvani) [inline Firmware/ControladorRobo/control.h:17]: Its good, a better explanation should be given when fixing issue #21

**Pull request merged**